### PR TITLE
Inicial setup "Training maquetado"

### PR DESCRIPTION
**Pull request by mattgle**
## Summary
Initial setup "Training maquetado"

## Trello Card
https://trello.com/c/m64Ng8SZ/11-setup-del-proyecto
## Screenshots
captura de pantalla 2018-10-18 a la s 16 51 36

### CODE REVIEW

File: maquetado/.gitignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+### Sass ###
+.sass-cache/
+.scss-cache/
+*.css.map
+*.sass.map
+*.scss.map
+*.css
+
+## System ##
+.DS_Store
```
Comment (henryzarza) [inline maquetado/.gitignore]: Add a break line

File: maquetado/application.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.title {
+    color: red;
+  }
```
Comment (henryzarza) [inline maquetado/application.scss]: Add a break line and the bracket should be aligned.

File: maquetado/application.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.title {
+    color: red;
```
Comment (henryzarza) [inline maquetado/application.scss]: Remember that the indentation should be of 2 spaces. Check this [style guide](https://github.com/Wolox/tech-guides/blob/master/frontend/docs/css-style-guide.md#indentation)

File: maquetado/index.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <h1 class="title">Hello World</h1> 
+</body>
+</html>
```
Comment (henryzarza) [inline maquetado/index.html]: Add a break line

**Review by henryzarza — CHANGES_REQUESTED** [review]

**Review by henryzarza — APPROVED** [review]
Good job 👍

**Review by SKOLZ — APPROVED** [review]

**Pull request merged**